src/Packagist/WebBundle/Resources/config/services.yml

```diff
@@ -1,7 +1,7 @@
 services:
     packagist.twig.extension:
         class: Packagist\WebBundle\Twig\PackagistExtension
-        arguments: [ @doctrine ]
+        arguments: [ '@doctrine' ]
```

Comment:
I'm used to use single quotes if double are not necessary, as PHP rules.

Do you want me to put single quote everywhere or to move back to double quotes?